schedule/storetemporaryschedules.go

```diff
@@ -116,7 +117,12 @@ func (store *Store) updateFixedShifts(ctx context.Context, tx *sql.Tx, scheduleI
 		return err
 	}
 
-	rawData, err = json.Marshal(data)
+	if len(rawData) > 0 {
```

Comment:
Do we need to be unmarshalling on [L109](https://github.com/target/goalert/pull/1476/files#diff-171bca7f32612fd22236d2f6213d0509513e843a62673c8f1e66b2eeb0688740R109) above since we do so inside of `Merge()`?

Reply:
Yes, because we need the struct populated (if `rawData` isn't empty) for the `apply` function that makes whatever necessary changes.